ethportal-api/src/types/content_key.rs

```diff
@@ -15,6 +15,11 @@ pub trait OverlayContentKey:
     fn content_id(&self) -> [u8; 32];
     /// Returns the bytes of the content key.
     fn to_bytes(&self) -> [u8; 33];
+    /// Returns the content key as a hex encoded "0x"-prefixed string.
+    fn to_hex_string(&self) -> String {
+        let hex = hex::encode(self.to_bytes());
+        format!("0x{hex}")
+    }
```

Comment:
Imo, if we're adding a `to_hex` / `to_hex_string` fn then we should also include the complementary `from_hex` / `from_hex_string` fn

Reply:
No problem

Reply:
Yup, good point